packages/react-native-bridge/android/react-native-bridge/build.gradle

```diff
@@ -81,43 +59,30 @@ dependencies {
 
     testImplementation "junit:junit:4.13"
 
+    def rnVersion = readReactNativeVersion('../../../../package.json', 'devDependencies')
+    println "react-native version for react-native-bridge: $rnVersion"
+
+    implementation "com.facebook.react:react-native:$rnVersion"
+    implementation "com.github.wordpress-mobile:react-native-svg:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-svg', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-video:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-video', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-linear-gradient:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-linear-gradient', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-slider:${extractPackageVersion('../../../react-native-editor/package.json', '@react-native-community/slider', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-get-random-values:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-get-random-values', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-masked-view:${extractPackageVersion('../../../react-native-editor/package.json', '@react-native-community/masked-view', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-gesture-handler:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-gesture-handler', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-screens:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-screens', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-safe-area-context:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-safe-area-context', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-reanimated:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-reanimated', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-prompt-android:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-prompt-android', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-webview:${extractPackageVersion('../../../react-native-editor/package.json', 'react-native-webview', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-clipboard:${extractPackageVersion('../../../react-native-editor/package.json', '@react-native-clipboard/clipboard', 'dependencies')}"
```

Comment:
I'm considering adding a constant `def packageJson = '../../../react-native-editor/package.json'` to simplify a bit the statements.

Reply:
I applied this change in https://github.com/WordPress/gutenberg/pull/39294/commits/c8b27cc971a33114ad2a4e68e8dc31527b4db27c.